code/scripts/article.js

```diff
@@ -0,0 +1,26 @@
+var articles = [];
+
+function Article (obj) {
+  this.place = obj.place;
+  this.guideUrl = obj.guideUrl;
+  this.body = obj.body;
+}
+
+// Define a method on the Article prototype
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+  $newArticle.find('h2').text(this.place);
+  $newArticle.find('address a').attr('href', this.guideUrl);
+  $newArticle.find('.content').html(this.body);
+
```

Comment:
//So far so good. this is roughly what I had when I did this part. once handlebars is added, it will clean this part up really nicely